shebangConvention.fsx

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env fsx
```

Comment:
let's not use fsx but dotnet fsi, fsx is too advanced for this repo (its most important benefit is performance, which we don't need here)

Reply:
Done